recipe/recipe.yaml

```
@@ -18,39 +18,29 @@ build:
 
 requirements:
   host:
-    - python >=${{ python_min }},<3.13
+    - python ${{ python_min }}.*
     - pip
-    - setuptools
+    - hatchling
   run:
     - python >=${{ python_min }},<3.13
-    - openai >=1.3
-    - diskcache
-    - termcolor
-    - flaml
-    - numpy >=1.17.0,<2
-    - python-dotenv
-    - tiktoken
-    - pydantic >=1.10,<3,!=2.6.0
-    - docker-py
-    - packaging
+    - autogen-core =0.6.4
```

```suggestion
    - autogen-core ={{ version }}
```

Should this just be dynamically updated to match the version?

---

Good catch - thanks for the suggestion.